### DISCUSSION TITLE
Blury screenshots in the diff folder

### DISCUSSION THREAD

**Discussion by Thamilalagan** [mjhea0/cypress-visual-regression#230] [General]
Hi All, 

Am using cypress-visual-regression for the past one month and found an issue in the screenshots taken that are stored in the diff folder. When I run the scripts for base version "npx cypress run --env type=base --spec cypress/e2e/visual-regression/" am able to get the screenshots ok whereas in the "node ./node_modules/cypress/bin/cypress run --env type=actual--spec cypress/e2e/visual-regression/" am getting the blury screenshots in the diff folder. Tried to put wait for upto 5 seconds but that didn't work out. Any idea how I can resolve the issue ?

**Comment by area73** [comment]
Hi @Thamilalagan, what do you mean by blurry?, are you getting differences running the same test without modifying the code?
If that is the case you need to make sure you are taking the snapshot after everything is loaded, for example some times the fonts take time to load so if you take the screenshot before all the CSS is loaded and parsed you may end up with differences, also take care about images loading, if you are loading images you have to make sure that the images are downloaded and in their final resolution (you may be using a lazy loading or something similar for images)

Anyway, if you still feel that this is an issue please feel free to provide us with a working example and we can take a look at it

I hope this could help

Thanks
